File: app/src/main/java/com/github/android/lvrn/lvrnproject/view/fragment/entitieslist/core/trashlist/TrashListFragment.java
package com.github.android.lvrn.lvrnproject.view.fragment.entitieslist.core.trashlist;

import com.github.android.lvrn.lvrnproject.view.fragment.entitieslist.EntitiesListWithSearchFragment;
import com.github.valhallalabs.laverna.persistent.entity.Note;

/**
 * @author Andrii Bei <psihey1@gmail.com>
 */

public interface TrashListFragment extends EntitiesListWithSearchFragment {

    String getSearchQuery();

    void showSelectedNote(Note note);
}

File: app/src/main/java/com/github/android/lvrn/lvrnproject/view/fragment/entitieslist/core/trashlist/TrashListFragment.kt
package com.github.android.lvrn.lvrnproject.view.fragment.entitieslist.core.trashlist

import com.github.android.lvrn.lvrnproject.view.fragment.entitieslist.EntitiesListWithSearchFragment
import com.github.valhallalabs.laverna.persistent.entity.Note

interface TrashListFragment : EntitiesListWithSearchFragment {

    override fun getSearchQuery(): String

    fun showSelectedNote(note: Note)
}